indy_node/server/request_handlers/read_req_handlers/get_attribute_handler.py

```diff
@@ -54,13 +55,13 @@ def get_attr(self,
                  did: str,
                  key: str,
                  attr_type,
-                 is_committed=True) -> (str, int, int, list):
+                 is_committed=False) -> (str, int, int, list):
```

Comment:
is_committed needs to be True for all `get_result` methods

Reply:
fixed